pages/welcome/add-profile-info.js

```diff
@@ -120,16 +128,32 @@ const AddProfileInfo = ({}) => {
               )}
 
               <Divider />
-
+              {showError ? (
+                <Grid container>
+                  <Grid item xs={12}>
+                    <Alert severity="error">
+                      <Stack>
+                        {showError.main}
+                        <Typography variant="bodySmall" error>
+                          {showError.sub}
+                        </Typography>
+                      </Stack>
+                    </Alert>
+                  </Grid>
+                </Grid>
+              ) : null}
               <Grid container justifyContent="center">
                 <Grid item xs={12} sm={8} md={6}>
                   <StyledFilePond
                     files={profilePicture}
                     allowReorder={false}
                     allowMultiple={false}
-                    maxFileSize="750KB"
+                    maxFileSize="5MB"
```

Comment:
I think an iPhone can take 20MB pictures these days.  If there's no special reason for 5MB, we should consider increasing this.  Ideally we downsize it in post-processing but for now I think it is okay to error towards more.